components/migrations.go

```diff
@@ -204,6 +204,16 @@ func (comp *migrationsComponent) Reconcile(ctx *cu.Context) (cu.Result, error) {
 		}
 	}
 
+	// add annotations to the job's pod template
+	jobTemplateAnnotations := map[string]string{
+		webhook.NOWAIT_MIGRATOR_ANNOTATION: "true",
+	}
+	if obj.Spec.Annotations != nil {
+		for k, v := range obj.Spec.Annotations {
+			jobTemplateAnnotations[k] = v
+		}
+	}
+
```

Comment:
Doing it in this order does technically allow the footgun of overriding the nowait annotation itself but I don't think anyone could do that by accident :)